trace/opencensus/trace/span_context.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""SpanContext encapsulates the current context within the request's trace."""
+
+from opencensus.trace.trace import generate_trace_id
```

Comment:
Can you change this to "from opencensus.trace import trace" to follow the "only import modules, not attributes" style guide?  Also, move this import below the standard library imports of logging and re?